setup.py

```diff
@@ -0,0 +1,6 @@
+# file required until PEP660 is correctly supported by setuptools
+# https://github.com/pypa/setuptools/issues/2816
+
+from setuptools import setup
+
+setup()
```

Comment:
We don't need setup.py with Poetry, on package build it will auto-generate a setup.py with metadata from pyproject.tomle.

Reply:
Don't we if a developer git clones and wants to run `pip install -e .`? https://github.com/python-poetry/poetry/issues/34

Reply:
Does poetry install -E <extra_name> -E <other_extra_nome> not work for you?

Reply:
No that only installs dev dependencies. Currently in https://github.com/python-poetry/poetry/issues/34 it is said that one cannot run the equivalent of `pip install -e .`.